### DISCUSSION TITLE
remove fail check for replay, make playback automatically pause at the last playback frame

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#29475] [Ideas]
I think this makes replay more like a (replay media?) player

first score in video is playing in multiplayer (no fail animation), so there are subsequent playback frames after fail
second score is playing in solo and saved, so there are no subsequent playback frames after fail

https://github.com/user-attachments/assets/1734900c-1d25-46af-ae13-255426b9dcf3

**Comment by peppy** [comment]
Dunno.

**Reply by cdwcgt** [reply]
The reason I'm making this proposal is that the current mechanism makes it impossible to replay the score which failed in multiplayer. I think whether the replayPlayer continues to play should be decided by the replay (frame) itself.
and also will not need to restart due to playing the fail animation

This also solves the problem of failed score in stable being imported into lazer, because they will not be marked as rank F.
But since they are failed score, they cannot continue to replay (no replay frames)

and for future, if will want player can continue play after failed (see https://github.com/ppy/osu/discussions/25921),
we should also hope that the replay can play normally but not fail in middle of replay and ignoring everything that follows

I hope you can consider
